features/aanschrijfwijze.feature

```diff
@@ -136,135 +141,622 @@ Functionaliteit: Als gemeente wil ik de juiste en consistente aanschrijfwijze va
+      | N           | In het      | Veld          | Henk      |                 |                     | Groenlo               | H. In het Veld-Groenlo             | VL VV GN-GP                   | geen adellijke titel/predikaat, voorvoegsel, partner geen voorvoegsel      |
+      | P           |             | Groenen       | Franklin  |                 | van                 | Velzen                | F. van Velzen                      | VL VP GP                      | geen adellijke titel/predikaat, partner voorvoegsel                        |
+      | V           | In het      | Veld          | Henk      |                 | van                 | Velzen                | H. van Velzen-In het Veld          | VL VP GP-VV GN                | geen adellijke titel/predikaat, voorvoegsel, partner voorvoegsel           |
+      | E           |             | Groenen       | Franklin  | R               |                     | Groenlo               | F. ridder Groenen                  | VL AT GN                      | adellijke titel, geen voorvoegsel                                          |
+      | N           |             | Groenen       | Franklin  | R               | van                 | Velzen                | F. ridder Groenen-van Velzen       | VL AT GN-VP GP                | adellijke titel, geen voorvoegsel, partner voorvoegsel                     |
+      | P           |             | Groenen       | Franklin  | R               |                     | Groenlo               | F. Groenlo                         | VL GP                         | adellijke titel, partner geen voorvoegsel                                  |
+      | V           | In het      | Veld          | Henk      | JH              | van                 | Velzen                | H. van Velzen-jonkheer In het Veld | VL VP GP-PK VV GN             | predikaat, voorvoegsel, partner voorvoegsel                                |
+
+  Rule: De aanschrijfwijze voor een persoon met partner met adellijke titel wordt als volgt samengesteld:
+        |                                                                                          | naamgebruik | aanschrijfwijze   | opmerking |
+        | geslachtsaanduiding "V", partner: geslachtsaanduiding "M", geen voorvoegsel              | P           | VL AP GP          | 1         |
+        | geslachtsaanduiding "V", geen voorvoegsel, partner: geslachtsaanduiding "M", voorvoegsel | N           | VL GN-AP VP GP    | 1         |
+        | geslachtsaanduiding "V", voorvoegsel, partner: geslachtsaanduiding "M", voorvoegsel      | V           | VL AP VP GP-VV GN | 1         |
+
+        1. adellijke titel partner wordt in vrouwelijke vorm opgenomen voor de achternaam (voorvoegsel + geslachtsnaam) van de partner
+          | titel | vrouwelijke vorm |
+          | graaf | gravin           |
+          | baron | barones          |
+          | prins | prinses          |
+
```

Comment:
Ik zou ook hier de onderliggende regels explicieter maken:

wanneer geslachtsaanduiding "V" en geslachtsaanduiding partner "M" en naam partner wordt gebruikt (P, N of V), wordt de adellijke titel van partner in vrouwelijke vorm opgenomen:

1. wanneer geslachtsaanduiding ≠ "V" ("M", "O" of leeg/null) wordt de adellijke titel van partner niet opgenomen
2. wanneer geslachtsaanduiding partner ≠ "M" ("V", "O" of leeg/null) wordt de adellijke titel van partner niet opgenomen
3. wanneer adellijke titel van partner geen vrouwelijke vorm heeft (ridder) wordt de adellijke titel van partner niet opgenomen
4. predikaat van partner wordt niet opgenomen

Reply:
Fijn, want dit was allemaal impliciet in de vorige versie. Ik kan mij voorstellen dat je dit met reverse engineering van de vorige versie van de feature niet kunt achterhalen.